src/IPM/callbacks.jl

```diff
@@ -147,17 +147,37 @@ function eval_lag_hess_wrapper!(
     n, p = size(Bk)
     m = size(kkt.jac, 1)
 
+    # Code for GPU support
+    if VT != Vector{T}
+        haskey(kkt.etc, :l_g) || (kkt.etc[:l_g] = VT(undef, m))
+        l_g = kkt.etc[:l_g]
+        copyto!(l_g, l)
```

Comment:
See function `get(dict, key, default)` to get a key if it exist, otherwise insert and return default. This avoids the extra key lookups.